### DISCUSSION TITLE
missing type definitions for typescript

### DISCUSSION THREAD

**Discussion by Ofer-Gal** [microsoft/fluentui#26120] [Q&A]
How do I figure the typescript missing type definitions for properties like:
e: SelectionEvents, data: OptionOnSelectData 
or (e: ComboboxBaseOpenEvents, data: ComboboxBaseOpenChangeData

They are shown in the properties of Select and Combo box  events but VC code does not know where to import from.
I did not find where to import them from.

Thanks

**Comment by vibraniumforge** [comment]
I agree. In the index.d.ts line 254, OptionOnSelectData is exported, but I can't import it. I can import the one above it, OptionGroupProps, but not OptionOnSelectData. 

The error is " Module '"@fluentui/react-components"' has no exported member 'OptionOnSelectData'.ts(2305) "

I am trying to avoid using an "any" in React Typescript on this line:
function handleSelectChange(data: OptionOnSelectData): void {

Is there a way to import this, or a better way to type an onchange event for the Fluent Dropdown onOptionSelect?
Thanks!

**Comment by khmakoto** [comment]
You should be able to type the handler as `const onOptionSelect: ComboboxProps['onOptionSelect']`.
